tools/run_tests/xds_k8s_test_driver/framework/xds_k8s_testcase.py

```diff
@@ -239,6 +240,43 @@ def assertSuccessfulRpcs(self,
             0,
             msg=f'Expected all RPCs to succeed: {failed} of {num_rpcs} failed')
 
+    @staticmethod
+    def diffAccumulatedStatsPerMethod(before, after):
+        """Only diffs stats_per_method, as the other fields are deprecated."""
+        diff = grpc_testing.LoadBalancerAccumulatedStatsResponse()
+        for method, method_stats in after.stats_per_method.items():
+            for status, count in method_stats.result.items():
+                count -= before.stats_per_method[method].result[status]
+                assert count >= 0, "Diff of count shouldn't be negative"
+                if count > 0:
+                    diff.stats_per_method[method].result[status] = count
+        return diff
+
+    def assertRpcStatusCodes(self, test_client: XdsTestClient, *,
+                             status_code: grpc.StatusCode, length: int,
+                             method: str) -> None:
+        # Sending with pre-set QPS for a period of time
+        before_stats = test_client.get_load_balancer_accumulated_stats()
+        logging.info(
+            'Received LoadBalancerAccumulatedStatsResponse from test client %s: before:\n%s',
+            test_client.ip, before_stats)
+        time.sleep(length)
+        after_stats = test_client.get_load_balancer_accumulated_stats()
```

Comment:
I guess we will keep using this call-sleep-call method then... (I thought you complaint about this approach.)

Reply:
I realized there was another comment hidden in a resolved conversation ^. This is optional though.

Reply:
The sleep command here isn't interesting. This was just a case of mistaken identity; this is not the sleep we had talked about :smile:. I do have a comment where the Empty/Unary switching is done, which is also where the Configure is done.